src/debugmode.lua

```diff
@@ -56,6 +65,46 @@ function Debug:draw()
 	end
 end
 
+-- Show invisible sensors.
+function Debug:drawSensors(player)
+	love.graphics.push("all")
+	love.graphics.setColor(0.5, 0.5, 0.5, 0.5)
+	love.graphics.setLineWidth(2/Settings.PARTSIZE)
+
+	local sensors = {}
+	local a, b, c, d = player.camera:getWorldBorder()
+	self.world.physics:queryBoundingBox(a, b, c, d, function(fixture)
```

Comment:
So we query for fixtures that collide with the camera a second time. We also do this for each player in player.lua. I'm not sure if it's better to loop over everything twice or cram everything into player.lua.